src/libs/fades.liq

```diff
@@ -68,15 +68,6 @@ def mkfade(~type="lin",~start=0.,~stop=1.,~duration=3.,~on_done={()},s) =
   fade
 end
 
-# Scale source during fading.
-# @category Source / Fade
-# @flag hidden
-def fade.scale(~id="fade.scale", x, s)
-  s = amplify(id=id, override=null(), x, s)
-  s = video.opacity(id=id, x, s)
-  s
-end
-
```

Comment:
Please leave `fade.scale` (but simply comment the line about `video.opacity`) and continue using it below. We ought to find a way to restore the support for video (and I think that I have ideas on this) and this will make things easier.

Reply:
I will do it for sake of brevity of this PR changes but I've never been a fan of doing implicit stuff with video. Most users would also be surprised by this I think. Things done on video should be explicit and be noted in the type as well.